include/grl/flatbuffer/KUKAiiwa.fbs

```diff
@@ -63,7 +63,9 @@ enum EControlMode : byte {
   POSITION_CONTROL_MODE=0,
   CART_IMP_CONTROL_MODE=1,
   JOINT_IMP_CONTROL_MODE=2,
-  NO_CONTROL=3
+  NO_CONTROL=3,
```

Comment:
@tdinesh KUKAiiwa.fbs is designed to match existing classes/enums in the FRI and Java APIs as perfectly as possible, so some of the changes in this pull request will need to be adapted to meet that requirement. Fortunately this enum seems to be the only item that deviated! 

Also, where is the code that handles these configurations on the JAVA side? It seems to be missing so these might be a no-op?